dspace-api/src/main/java/org/dspace/authorize/service/ResourcePolicyService.java

```diff
@@ -66,26 +67,60 @@
     public void removePolicies(Context c, DSpaceObject o, String type) throws SQLException, AuthorizeException;
 
     public void removeDsoGroupPolicies(Context context, DSpaceObject dso, Group group)
-        throws SQLException, AuthorizeException;
+            throws SQLException, AuthorizeException;
```

Comment:
there are many small formatting changes in the following lines that are not necessary. It would be great if you can undo them